provider/oauth2/views.py

```diff
@@ -8,13 +8,11 @@
 
 from provider import constants
 from provider.oauth2.backends import BasicClientBackend, RequestParamsClientBackend, PublicPasswordBackend
-from provider.oauth2.forms import AuthorizationCodeGrantForm
-from provider.oauth2.forms import AuthorizationRequestForm, AuthorizationForm
-from provider.oauth2.forms import PasswordGrantForm, RefreshTokenGrantForm
+from provider.oauth2.forms import (AuthorizationCodeGrantForm, AuthorizationRequestForm, AuthorizationForm,
+                                   PasswordGrantForm, RefreshTokenGrantForm, ClientCredentialsGrantForm)
 from provider.oauth2.models import Client, RefreshToken, AccessToken
 from provider.utils import now
-from provider.views import AccessToken as AccessTokenView, OAuthError, AccessTokenMixin
-from provider.views import Capture, Authorize, Redirect
+from provider.views import AccessToken as AccessTokenView, OAuthError, AccessTokenMixin, Capture, Authorize, Redirect
```

Comment:
This (i.e., `import AccessToken as AccessTokenView`) is what threw me off earlier. It's especially strange considering that a second `AccessTokenView` is being defined in this module.

I understand wanting to be as surgical as possible when it comes to touching this codebase. However, since you've fixed the imports here, would you be opposed to also fixing this so that `AccessToken` isn't renamed on import?